azext_iot/central/providers/device_provider.py

```diff
@@ -225,7 +225,35 @@ def get_component_command_history(
             command_name=command_name,
             central_dns_suffix=central_dns_suffix,
         )
+        
+    def run_manual_failover(
+        self,
+        device_id: str,
+        ttl_minutes:int=None,
```

Comment:
We should have defaulting to the same value we specify in the help text, i.e. this should be 30 not None